decayAmplitude/isobarDecayTopology.cc

```diff
@@ -875,3 +860,43 @@ isobarDecayTopology::findIsobarBoseSymVertices() const
 	}
 	return boseSymVertices;
 }
+
+/**
+ * Returns list with all possible permutations of all groups of particles
+ */
```

Comment:
`/*` -> `//`

Also, this function is very hard to understand, some comments maybe?